registry/registry.go

```diff
@@ -50,14 +42,17 @@ var (
 
 // Registration information.
 type Registration struct {
-	Key      string `json:"key"`
-	Address  string `json:"address"`
-	Registry string `json:"registry"`
+	Key         string   `json:"key"`
+	Address     string   `json:"address"`
+	Registry    string   `json:"registry"`
+	Annotations []string `json:"annotations"`
 }
 
 // String descritpion of registration.
 func (r *Registration) String() string {
-	return fmt.Sprintf("key: %v, address: %v, registry: %v", r.Key, r.Address, r.Registry)
+	sort.Strings(r.Annotations)
```

Comment:
[NIT] Do we need to sort annotations every time if they're sorted on entry into the registry?